build.gradle

```diff
@@ -360,12 +359,6 @@ task switchGoogleServicesCommcare(type: Copy) {
     description = 'Makes build use normal google-services.json file'
     println project.ext.COMMCARE_APP_ID
 
-    /**
-     * This is an awkward workaround for a Windows build error that occurs when you try
-     * to copy something into the root directory (containing the .gradle folder). Don't
-     * change it without testing on a windows machine
-     */
-    outputs.files.setFrom(file("$projectDir/google-services.json"))
```

Comment:
@ctsims I removed this because it was causing issues. Can you check to see if Gradle 3.1 fixed the issue for you on Windows?